src/store/middleware.js

```diff
@@ -0,0 +1,31 @@
+import { RECEIVE_MESSAGE, setActiveChat } from "./Chat/index";
+
+
+export const logger = store => next => action => {
```

Comment:
this is so bizzarre `store => next => action => {`

Reply:
this is from the redux docs

functions that take functions meman

Reply:
👍